requirements.in

```diff
@@ -9,7 +9,7 @@ django-cache-url>=1.0.0
 django-countries>=4.1
 django-mptt>=0.7.1
 django-payments>=0.11.0,<0.12a0
-django-prices>=0.4.11,<0.5a0
+django-prices>=0.4.11
```

Comment:
Should we drop the other "lower than" guards for consistency?

Reply:
Didn't want to remove guards I wasn't sure about. I will do it if there is no reason for specific api freeze anywhere.
What about Pillow library, is there any reason behind 4.0.0 upper bound?

Reply:
The `<0.x` guards predate `pip-compile`. I think now that we do explicit version bumps it does not make sense to have them unless they are followed with a comment explaining the reason (ie. "# see issue #0000").